static/src/stylesheets/module/content-garnett/_media.head.scss

```diff
@@ -52,3 +52,16 @@
         background: $brightness-20;
     }
 }
+
+/* Styling fix for special topic label */
+.content--media {
+    header {
+        .content__labels.content__labels--immersive {
+            background: none;
+            .badge-slot {
+                margin-bottom: 16px;
```

Comment:
Is this taking care of point 2?

Reply:
The banner has different behaviour at smaller breakpoints. But Akemi was happy with this implementation both above and below the change

Reply:
The text is intended to sit *below* the header rather than in it